typescript/packages/jumble/integration/utils.ts

```diff
@@ -39,3 +39,68 @@ export const login = async (page: Page) => {
   button = await page.$("button");
   await button!.click();
 };
+
+export const addCharm = async () => {
+  const space = `ci-${Date.now()}-${
+    Math.random().toString(36).substring(2, 15)
+  }`;
+  const process = new Deno.Command("deno", {
+    args: [
+      "task",
+      "start",
+      "--space",
+      space,
+      "--recipeFile",
+      "recipes/simpleValue.tsx",
+      "--cause",
+      "ci",
+      "--quit",
+      "true",
+    ],
+    env: {
+      "TOOLSHED_API_URL": "http://localhost:8000/",
+    },
+    cwd: "../common-cli",
+    stdout: "inherit",
+    stderr: "inherit",
+  });
+
+  const { code } = await process.output();
+  if (code !== 0) {
+    throw new Error(`Failed to add charm: ${code}`);
+  }
+
+  return {
+    charmId: "baedreic5a2muxtlgvn6u36lmcp3tdoq5sih3nbachysw4srquvga5fjtem",
+    space,
+  };
+};
+
+export const inspectCharm = async (space: string, charmId: string) => {
+  const process = new Deno.Command("deno", {
```

Comment:
As an alternative to calling the CLI via shelling out, since this is all in Deno, we could import whatever the CLI's "main" execution function is e.g.

```ts
import { run } from "@commontools/cli"

run({ space, charmId })
```

Or any `common-charm` etc. running, just as a future option